assets/html/index.html

```diff
@@ -1 +1,22 @@
-
+<html>
+  <head>
+    <title>Challenge One</title>
+  </head>
+  <body>
+    <h1 align="center">This is Chris Featley's page</h1>
+    <p align="center">This should be a photo of Chris</p>
+    <!--I realise I need to actually add a photo here and then do img src -->
+    <p align="center">My Hobbies</p>
+    <ul align="center">
+      <li>hobby 1</li>
+      <li>hobby 2</li>
+      <li>hobby 3</li>
+    </ul>
+    <p align="center">More about me</p>
+    <a href="www.facebook.com" align="center">Link to my facebook</a>
+    <br />
+    <a href="www.twitter.com" align="center">Link to my twitter</a>
```

Comment:
these don't link correctly

Reply:
links should work now as included the http://